Silicon/CommonSocPkg/Include/Library/WatchDogTimerLib.h

```diff
@@ -44,14 +45,26 @@ WdtDisable (
 
 
 /**
-  Clear WDT timer flags.
+  Clear WDT TCC flags in scratchpad
 
-  @param[in] Flags             The timer flags that would be cleared.
+  @param[in] Flags             The scratchpad flags that would be cleared.
 
 **/
 VOID
 EFIAPI
-WdtClearFlags (
+WdtClearTccScratchpad (
+  IN  UINT32  Flags
+  );
+
+/**
+  Set WDT TCC flags in scratchpad
+
+  @param[in] Flags             The scratchpad flags that would be set.
+
+**/
+VOID
+EFIAPI
+WdtSetTccScratchpad (
```

Comment:
WdtDogTimerLib is for common usage, and it should not be specific to TCC.
Please keep the API interface generic and use the flag to indicate TCC use cases.

Reply:
rename to WdtSetScratchpad() to keep common usage and focus on scratchpad only